### DISCUSSION TITLE
NCNN framvork build whith  mingw on Windows

### DISCUSSION THREAD

**Discussion by SeyranGrig** [Tencent/ncnn#3153] [General]
Is it possible to build NCNN framework using mingw on Windows?

**Comment by magicse** [comment]
> Is it possible to build NCNN framework using mingw on Windows?

Hi, yes it possible.
I compiled  ncnn with mingw64+openCV+Vulkan support for windows 7 and it work well. Also compiled binaries work well on windows 10
